### ISSUE TITLE
Random/Practice Mode

### ISSUE THREAD

**Issue by emckee2006** [Compsciler/Notwordle#30]
This is more of a feature request then an issue, but it would be nice to have the ability to play a random puzzle in addition to the daily. I know streamers who would enjoy playing this game on their stream, for example, but they just can't do that because people in chat spoil daily games instead of working together to solve the puzzle. The main reason, though, would be just to be able to enjoy this game more frequently :) Thanks in advance :)

**Comment by EmilyV99** [comment]
Yeah, only being able to play once a day kinda sucks. A random mode would be great!
